### PR TITLE
Fix Bug 1276217 - Visual regressions in default newsletter embed forms 

### PR DESCRIPTION
**Pull request by RCMainak**
## Description
Increased the input email width for mobile view. Decreased the .span_narrow of form-title by 1 unit & Increased the .span_narrow of form-submit by 1 unit for tablet.
## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1276217
## Testing
Checked on only the contact page using the built-in options in Responsive Design Mode of Firefox. 
## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: media/css/sandstone/sandstone-resp.less [hunk 1]
```diff
@@ -899,7 +899,7 @@ nav.menu-bar {
     }
 
     input[type=email] {
-        width: 80%;
+        width: 92%;
```
Comment (craigcook) [inline media/css/sandstone/sandstone-resp.less]: I think you can use the `.border-box;` mixin here (which sets `box-sizing: border-box;`, the mixin just includes the vendor prefixes for browsers that need them) and set this width to 100%.

Comment (RCMainak) [inline media/css/sandstone/sandstone-resp.less]: Let me take a look into it.

**Comment by RCMainak** [comment]
Wow, that was a fun fix. It's a shame I didn't checked the corner cases. Thank you for enlightening me. Hope it's okay now :smile:

**Comment by craigcook** [comment]
Looks good. Well done! :newspaper_roll:

**Pull request merged**